kratos/python_scripts/response_functions/response_function_interface.py

```diff
@@ -26,6 +26,9 @@ def RunCalculation(self, calculate_gradient):
     def Initialize(self):
         pass
 
+    def UpdateDesign(self, updated_model_part):
```

Comment:
```suggestion
    def UpdateDesign(self, updated_model_part, variable
):
```

Reply:
Hmm... I am wondering how did the CI pass without this??

Reply:
would it put all the calling arguments into a tuple for the single argument in the interface function? Not sure